docs/ug/configFiles.md

```diff
@@ -70,13 +70,15 @@ Optionally, you can use an `author-config.csv` (which should be in the same dire
 |-------------|-------------|
 | Repository's Location | Same as `repo-config.csv`. Default: all the repos in `repo-config.csv` |
 | Branch | The branch to analyze for this author, e.g., `master`. Default: the author will be bound to all the repos in `repo-config.csv` that has the same repo's location, regardless of branch. |
-| Author's GitHub ID {{ mandatory }} | Username of the target author's profile on GitHub, GitLab or Bitbucket, e.g.`JohnDoe`. |  
+| Author's Git Host ID<sup>**</sup> {{ mandatory }} | Username of the target author's profile on GitHub, GitLab or Bitbucket, e.g.`JohnDoe`. |  
 | Author's Emails<sup>*</sup> | Associated emails of the author. For GitHub users, this can be found in your [GitHub settings](https://github.com/settings/emails). |
 | Author's Display Name | The name to display for the author. Default: author's username. |
 | Author's Git Author Name<sup>*</sup> | The meaning of _Git Author Name_ is explained in [_A note about git author name_](#a-note-about-git-author-name). |
 | Ignore Glob List<sup>*</sup> | Files to ignore for this author, in addition to files ignored by the patterns specified in `repo-config.csv`. The path glob syntax is the same as that of Ignore Glob List in `repo-config.csv`. |
 
 <sup>* **Multi-value column**: multiple values can be entered in this column using a semicolon `;` as the separator.</sup>
+</br>
+<sup>** **Backward compatibility**: `Author's Git Host ID` can be replaced with `Author's GitHub ID`.</sup>
```

Comment:
I would write it as something like 
> For backward compatibility, `Author's GitHub ID` is also accepted as the header in place of `Author's Git Host ID`

and remove the bolded label

Reply:
I see, I had it in bold just to follow the previous line XD

Reply:
btw just to double-check, `**` is not used right? Or will some other symbols be preferred to make it less ambiguous with the single `*`?

Reply:
Maybe a `1` subscript? Or I can use `+` like the other table above?

Reply:
I think we should reserve `+` for a single meaning (overridable column in this case). Maybe something like `#` or `&`?

Reply:
Okay, I'll change it to `#` then.